test/query.ts

```diff
@@ -206,6 +206,18 @@ describe('Query', () => {
   });
 
   describe('filter', () => {
+    it('should issue a warning when a Filter instance is not provided', done => {
```

Comment:
It might be nice to make this validate that the warning only happens once, but since gax probably also tests that, it's probably not a big issue.

Reply:
Done